dockers/docker-teamd/teamd.sh

```diff
@@ -6,7 +6,7 @@ function start_app {
     rm -f /var/run/teamd/*
     if [ "$(ls -A $TEAMD_CONF_PATH)" ]; then
         for f in $TEAMD_CONF_PATH/*; do
-            teamd -f $f -d
+            teamd -f $f -d -N -o -w
```

Comment:
does it mean it is already warm reboot? can we do without warm reboot?

Reply:
for normal boot, we do not want enable warm reboot option here.

Reply:
but when we want to enable  warm reboot it would be already late
We need to decide how to enable warm reboot logic after teamd start.

Reply:
Could similar check like that for swss be done? https://github.com/Azure/sonic-buildimage/blob/master/files/scripts/swss.sh#L30